### PR TITLE
 ✨ [#246] Add is_standaard_adres for DigitaalAdres

### LINKED ISSUE #246: Store default address per 'soort digitaal adres'
### Thema / Theme

Klantinteracties API

### Acceptance criteria
* At any time only a single digital address can be default per 'soort digitaal adress'
* It is possible to set the default
* It is possible to change the default

### Toegevoegde waarde / Added value

In case there are multiple digital addresses of a certain type, it is needed to know which one to use for a certain communication. The existing 'voorkeur digitaal adress' is not enough as that may be an address which cannot be used for a certain communication

### Aanvullende opmerkingen / Additional context

_No response_

### PR DESCRIPTION
**Pull request by stevenbal**
Fixes #246 

**Changes**

* Add is_standaard_adres for DigitaalAdres

### CODE REVIEW

**Review by bart-maykin — COMMENTED** [review]
You should also change the validation for the `voorkeur digitaal adress` to only allow these default addresses. Because I don't see a scenario where this would be wanted behavior.

**Comment by stevenbal** [comment]
@bart-maykin @joeribekker hmm you're right, though upon looking at this again, I don't know if what I implemented here makes sense to begin with actually, because with the current logic there can only be three default addresses in total (one for each type). Should I add the `betrokkene` and `partij` to the condition in the unique constraint as well?

**Comment by joeribekker** [comment]
refinement: So, Partij should be included in the unique check.

We should allow something like this, where preference is not necessarily a default option (this should be enforced by the client app if needed).

default = per type, per party
preference = overall preferred option, for a party

a. email: john@gmail.com (default, preference)
b. email: john@maykin.com
c. phone: 01234567890 (default)
d. phone: 09876544321

**Comment by stevenbal** [comment]
@annashamray this PR is ready for review.

I was working on splitting DigitaalAdres (https://github.com/maykinmedia/open-klant/pull/277) into two separate resources (one linked to Partij and one to Betrokkene), but as we talked about on slack we should probably discuss this next week, since it would be a breaking change

File: src/openklant/components/klantinteracties/models/digitaal_adres.py [hunk 1]
```diff
@@ -55,6 +62,25 @@ class DigitaalAdres(APIMixin, models.Model):
 
     class Meta:
         verbose_name = _("digitaal adres")
+        constraints = [
+            models.UniqueConstraint(
```
[the remark this answers is not in the document]
Comment (stevenbal) [reply src/openklant/components/klantinteracties/models/digitaal_adres.py:66]: It should only be used if partij is set, I'll add explicit validation for it

**Review by bart-maykin — APPROVED** [review]

**Pull request merged**